frontend/src/routes.js

```diff
@@ -409,7 +409,11 @@ const routes = [
     children: [
       {
         path: '',
-        element: <Catalog />
+        element: isModuleEnabled(ModuleNames.CATALOG) ? (
```

Comment:
Instead of making the Catalog the default and then Organization. Could we define the `Environments` view as default always? 
- Environments better than Organizations because they are more used, they are the AWS accounts-link
- better to have some core as default and then forget about the modules in the defaults
- better to have environment as default because the Catalog performs queries to OpenSearch which at enterprise scale can be costly => let's save soooome money